index.html

```diff
@@ -21,37 +21,49 @@
           <option value="LU">language of random</option>
         </select>
       </p>
+<!-- This is pretty unclear for a newbie (like me) .. should break it up and provide more detail (with comments)? -->
       <form action="/action_page.php">
-        <label for="fname">
+        <label for="inputtext">
           <span lang="en">Message:</span>
           <span lang="de">Nachricht:</span>
           <span lang="cn">讯息:</span>
           <span lang="es">Mensaje:</span>
           <span lang="LU">QWERTYUIOP:</span>
         </label>
-        <input type="text" id="fname" name="fname"><br><br>
-        <label for="sname">
-          <span lang="en">Secret:</span>
+<!--        <input type="inputtext" id="inputtext" name="inputtext"><br><br> -->
+        <textarea id="inputtext" name="inputtext" rows="4" cols="50">
+        Enter message to be de/encrypted.
+        </textarea><br><br>
+        
+        <label for="key">
+          <span lang="en">Secret Key:</span>
           <span lang="de">Schlüssel:</span>
           <span lang="cn">秘密:</span>
           <span lang="es">Secreto:</span>
           <span lang="LU">TyPe LeTtErS:</span>
-        </label>
-        <input type="text" id="sname" name="sname"><br><br>
-        <label for="lname">
+        </label>        
+        <input type="key" id="key" name="key"><br><br>
+
+        <label for="outputtext">
           <span lang="en">Decoded Message:</span>
           <span lang="de">Entschlüsselte Nachricht:</span>
           <span lang="cn">解码后的消息:</span>
           <span lang="es">Mensaje Decodificado:</span>
           <span lang="LU">SUPPOSED MESSAGE:</span>
         </label>
-        <input type="text" id="lname" name="lname"><br><br>
+<!--        <input type="text" id="outputtext" name="outputtext"><br><br> -->
+        <textarea id="inputtext" name="inputtext" rows="4" cols="50">
```

Comment:
You need to make the `id` unique. And since you are using `inputtext` already in https://github.com/video-game-coding-club/crypto-webpage/pull/63/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R34 you need to pick some other name here.

Reply:
Oops,... sorry about taking so long to fix.

Reply:
Thanks! No worries :smile: